lib/task-data/schemas/linux-command.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "rackhd-task-schema.json",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Linux Command in Microkernel",
+    "description": "The schema for linux command job",
+    "describeJob": "Job.Linux.Commands",
+    "definitions": {
+        "SimpleCommandItem": {
+            "description": "The command line including the arguments",
+            "type": "string",
+            "minLength": 1,
+            "readonly": true
+        },
+        "AdvanceCommandItem": {
+            "type": "object",
+            "properties": {
+                "catalog": {
+                    "description": "Indicate the command result should be stored into catalog and specify the parameter while storing catalog",
+                    "type": "object",
+                    "properties": {
+                        "source": {
+                            "description": "The source where the catalog data is from",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "format": {
+                            "description": "The catalog format, such as 'json', 'raw', .etc",
+                            "type": "string",
+                            "enum": ["json", "raw"]
+                        }
+                    },
+                    "required": ["source"],
+                    "additionalProperties": false
+                },
+                "command": {
+                    "$ref": "#/definitions/SimpleCommandItem"
+                },
+                "retries": {
+                    "description": "The total retry times if the command execution fails",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "downloadUrl": {
+                    "description": "Specify the download URL (relative to RackHD southbound API server) if the command needs to download some data or script from RackHD before execution",
+                    "type": "string",
+                    "pattern": "^/[a-zA-Z0-9+-.#?=\\\/]+"
```

Comment:
As I said in  the description, this is a URL  that relative to RackHD southboud API server, below is an example:
https://github.com/RackHD/on-tasks/blob/ed2a5e49998a04da0901319f015faec841c19bba/lib/task-data/tasks/catalog-smart.js#L13